src/ClientManager/Travely.ClientManager.Service/Mappers/PreferenceProfile.cs

```diff
@@ -0,0 +1,20 @@
+﻿using AutoMapper;
+using Google.Protobuf.WellKnownTypes;
+using System;
+using Travely.ClientManager.Abstraction.Entity;
+using Travely.ClientManager.Service.Protos;
+
+namespace Travely.ClientManager.Service.Mappers
+{
+    public class PreferenceProfile : Profile
+    {
+        public PreferenceProfile() 
+        {
+            CreateMap<Preference, PreferenceModel>()
+                 .ForMember(dest => dest.CreatedDate, opt => opt.MapFrom(src => DateTime.SpecifyKind(src.CreatedDate, DateTimeKind.Utc).ToTimestamp()));
```

Comment:
Why we need this Converted value instead of CreatedDate?

Reply:
Hov jan in PreferenceModel (GRPC model) createdDate type is a Timestamp Class , 
in Preference (DB entity) createdDate type is Datetime, 
thats why we use mapping here

Reply:
👍